content/integrations/wmi.md

```diff
@@ -1,15 +1,134 @@
+
+If you assign the new metric a category of My_New_Metric, the WMI path will be 
+`\\<ComputerName>\ROOT\CIMV2:Win32_PerfFormattedData_My_New_Metric`
+
+If the metric isn't showing up in WMI, try running `winmgmt /resyncperf` to force the computer to reregister the performance libraries with WMI.
+
+
+# Configuration
+
+1.  Click the **Install Integration** button on the WMI Integration Tile. 
+2.  Open the Datadog Agent Manager on the Windows server.
+3.  Edit the `Wmi Check` configuration.
+  
+        init_config:
+
+        instances:
+          - class: Win32_OperatingSystem
+            metrics:
+              - [NumberOfProcesses, system.proc.count, gauge]
+              - [NumberOfUsers, system.users.count, gauge]
+
+          - class: Win32_PerfFormattedData_PerfProc_Process
+            metrics:
+              - [ThreadCount, proc.threads.count, gauge]
+              - [VirtualBytes, proc.mem.virtual, gauge]
+              - [PercentProcessorTime, proc.cpu_pct, gauge]
+            tag_by: Name
+
+          - class: Win32_PerfFormattedData_PerfProc_Process
+            metrics:
+              - [IOReadBytesPerSec, proc.io.bytes_read, gauge]
+            tag_by: Name
+            tag_queries:
+              - [IDProcess, Win32_Process, Handle, CommandLine]
+
+    Note that the default configuration uses the filter clause to limit the metrics pulled. Either set the filters to valid values or remove them as shown above to collect the metrics.
+
+    The metrics definitions include three components: class property in WMI, metric name as it appears in Datadog, and the metric type. 
+
+4.  Restart the agent.
+
+## Configuration Options
+
+Each WMI query has 2 required options, `class` and `metrics` and six optional options, `host`, `namespace`, `filters`, `tag_by`, `constant_tags` and `tag_queries`.
+
+`host` is the optional target of the WMI query, `localhost` is assumed by default if you set this option, make sure that Remote Management is enabled on the target host [see here](https://technet.microsoft.com/en-us/library/Hh921475.aspx) for more information.
+
+`class` is the name of the WMI class, for example `Win32_OperatingSystem` or `Win32_PerfFormattedData_PerfProc_Process`. You can find many of the standard class names on the [MSDN docs](http://msdn.microsoft.com/en-us/library/windows/desktop/aa394084.aspx).
```

Comment:
Should we arrange these option explanations in the order listed in line 71? So class and metrics would be above host?